components/identity-event/org.wso2.carbon.identity.event/src/main/java/org/wso2/carbon/identity/event/IdentityEventConstants.java

```diff
@@ -200,6 +200,7 @@ private Event(){}
         public static final String POST_GENERATE_SMS_OTP = "POST_GENERATE_SMS_OTP";
         public static final String POST_VALIDATE_SMS_OTP = "POST_VALIDATE_SMS_OTP";
         public static final String SESSION_EXTENSION = "SESSION_EXTENSION";
+        public static final String POST_JIT_USER_UPDATE = "POST_JIT_USER_UPDATE";
```

Comment:
why only the update event is considered? what about the create event?

Reply:
This handler gets triggered even if the user already provisioned or not. So decided to fire the event from here. The event name might confuse. So, changed it from `POST_JIT_USER_UPDATE` to `JIT_USER_SET_CLAIMS`. Fixed in https://github.com/wso2/carbon-identity-framework/pull/4612/commits/a4e3a9878fea0d2b572975f81dffafd696fbe36c